packages/commerce-sdk-react/src/auth/storage/cookie.ts

```diff
@@ -24,10 +24,45 @@ export class CookieStorage extends BaseStorage {
     }
     set(key: string, value: string, options?: Cookies.CookieAttributes) {
         const suffixedKey = this.getSuffixedKey(key)
-        Cookies.set(suffixedKey, value, {
-            ...getDefaultCookieAttributes(),
-            ...options
-        })
+        const defaultAttributes = getDefaultCookieAttributes()
+
+        const cookieOptions = {
+            ...defaultAttributes,
+            ...options,
+            expires: this.convertSecondsToDate(options?.expires)
+        }
```

Comment:
I don't think that this is the right place to making tis fix. Expires is a date and we are using seconds (ttl). We should be converting the ttl in seconds to a date before calling set. 

```
// auth/indx.ts line 381
this.set(refreshTokenKey, res.refresh_token, {
        expires: res.refresh_token_expires_in // This needs to be updated here.
    })
```